tests/data/raw/xmatch_expected/xmatch_correct_002_005.csv

```diff
@@ -0,0 +1,31 @@
+,ss_id,xmatch_id,dist
```

Comment:
How are these files generated?

Reply:
They were generated using a brute-force approach that computes the distances between the pairs of points formed between `small_sky` and `small_sky_xmatch`. 

Of those points, those that are within the `min_radius_arcsec` and `radius_arcsec` are kept and sorted by ascending order of separation. Finally, we select up to `n_neighbors`.